iml-services/iml-device/Cargo.toml

```
@@ -18,8 +20,9 @@ tokio = { version = "0.2", features = ["macros"] }
 tokio-util = { version = "0.3", features = ["codec"] }
 tracing = "0.1"
 tracing-subscriber = "0.2"
-
 warp = "0.2"
+diesel = "1.4.4"
```

```suggestion
diesel = { version = "1.4", default_features = false, features = ["postgres", "r2d2", "chrono", "serde_json"] }
```

This needs default_features disabled, as it reduces compile time for diesel by 40 seconds.